tests/integration/tests/test-create-baseline.js

```diff
@@ -0,0 +1,27 @@
+/*
+  This file is used to create a baseline image for a webpage.
+*/
+
+"use strict";
+
+
+describe("Firefox Monitor homepage", function() {
+    this.retries(2);
+
+    beforeEach(function() {
+      browser.url("http://localhost:6060/");
```

Comment:
Should this be localhost:6060 or just `/` (since `baseUrl` is `baseUrl: "http://localhost:6060",`)?

Reply:
Should just be ```/```